proxy/pkg/egress/egress_test.go

```diff
@@ -36,9 +40,21 @@ import (
 func BenchmarkMatch(b *testing.B) {
 	lager.Initialize("", "DEBUG", "",
 		"size", true, 1, 10, 7)
-
-	gopath := os.Getenv("GOPATH")
-	os.Setenv("CHASSIS_HOME", gopath+"/src/github.com/apache/servicecomb-mesher")
+	chassis := []byte(`
+cse:
```

Comment:
Do we still need to use cse setting here?

Reply:
yes, it is not changed yet, will support servicecomb prefix in future release